gulp/tasks/upload-preview.js

```diff
@@ -0,0 +1,17 @@
+(function() {
+    'use strict';
+
+    var gulp = require('gulp'),
+        gitUtils = require('../util/gitUtils'),
+        childProcess = require('child_process'),
+        previewDomain = 'ux-test.edx.org',
+        previewSiteDir = '_preview_site';
+
+    gulp.task('upload-preview', function() {
+        var branch = gitUtils.currentBranch();
+        childProcess.execSync(
+            'aws s3 sync ' + previewSiteDir + ' s3://' + previewDomain + '/' + branch
+        );
+        console.log('Preview site ready at http://' + previewDomain + '/' + branch);
+    });
+}());
```

Comment:
@andy-armstrong do we really need the remove and upload preview code in separate files? The require includes at the top of the file are almost identical. It just seems a little intense to have 1 gulp task per file as opposed to a collection of related tasks. Thoughts?